materialize-postgres/sqlgen_test.go

```diff
@@ -95,3 +96,54 @@ func TestTruncatedIdentifier(t *testing.T) {
 		})
 	}
 }
+
+func TestLoadQueryNoFlowDocumentTemplate(t *testing.T) {
```

Comment:
nit: is this left over from earlier initial testing?  Noting that other connectors don't have it, presumably because they're covered via common fixtures?